packages/site/src/providers/AtlantisPreviewEditorProvider.tsx

```diff
@@ -360,23 +360,24 @@ const skeletonHTML = (theme: Theme, type: "web" | "mobile") => {
 <head>
 <style>
 html,body,#root {
-  height: 100%;
-  width: 99%;
+  height: 100vh;
+  width: 100vw;
   display: flex;
   align-items: center;
   justify-content: center;
   flex-direction: column;
-  min-height: 200px;
+  min-height: 280px;
+  margin: 0;
```

Comment:
This, combined with the vh/vw above, let the contents use the full defined space as needed